### ISSUE TITLE
Add a callout for registry auth info

### ISSUE THREAD

**Issue by ps2goat** [docker/docker.github.io#5697]
File: [get-started/part4.md](https://docs.docker.com/get-started/part4/), CC @johndmulhausen 

# TL;DR; 
My private registry auth info had to be passed to the swarm when following the instructions on part 4.  I had to use `docker stack deploy -c docker-compose.yml --with-registry-auth getstartedlab` to pass the registry info to my swarm so they could independently pull down the image to run.

# Repro steps
I haven't tested this thoroughly, so I don't know all the ways to reproduce the issue.  Here is what I did:

In Part 2, I actually used AWS ECR as my registry, and I got that working following the instructions [here](https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-basics.html#use-ecr).  I had created the repo directly in ECR, then through the AWS CLI, I got the docker login credentials and registered it, tagged/renamed my image as required to get it into the ECR repo.

I was able to push the image to ECR.  I then deleted the image locally and pulled it down dynamically when starting the service.

However, when I got to part 4, I was unable to get the image to load by running `docker stack deploy -c docker-compose.yml getstartedlab`.  All services were reporting that there was "No such image: xxxxxxxxx", where xxxxxxxx = the start of my ECR repo name.

After searching, I found [this](https://github.com/moby/moby/issues/31357#issuecomment-283392083), which worked for me: `docker stack deploy -c docker-compose.yml --with-registry-auth getstartedlab`

It would be nice to have that called out in the getting started guide, the same way that things such as reverse proxy, port reservations, etc., are called out.

**Comment by ps2goat** [comment]
This is similar to issue 4107, but it is on a different page. This is why I decided to open a new issue.
